src/webview/js/js.js

```diff
@@ -926,9 +926,25 @@ const handleLongPress = (target: Element) => {
     return;
   }
 
+  const targetType = target.matches('.header')
+    ? 'header'
+    : target.matches('a')
+    ? 'link'
+    : 'message';
```

Comment:
Prettier has some kind of bug affecting nested `?:` operators. Use `prettier-ignore`, and give it a sensible indentation instead :slightly_smiling_face: 

Search the codebase for "ternary" to find a couple of other spots as examples.

Reply:
Fixed.